TensorFlowSharp/OperationsExtras.cs

```diff
@@ -31,7 +31,11 @@ TFOutput ReduceDims (TFOutput input, TFOutput? axis = null)
 			if (shape.IsFullySpecified) {
 				// The python code distinguishes between tensor and sparsetensor
 
-				return this.Const (shape.ToIntArray (), TFDataType.Int32);
+				var array = new int [shape.NumDimensions];
+				for (int i = 0; i < array.Length; i++)
+					array [i] = i;
+
```

Comment:
I would appreciate @migueldeicaza and @cesarsouza to review this changes. I returned initial implementation of ReduceDims, because changes introduced in [this ](https://github.com/migueldeicaza/TensorFlowSharp/commit/440066542a35fa62a013e5767b5d7f7ee47af56d#diff-9b54c6b41f47f0f34e4cbc0691cf1818) commit broke Should_ReduceMean() C# test.

My concern is whether we need to move this implementation to Shape.ToIntArray() method or leave here